src/CMakeLists.txt

```diff
@@ -250,6 +248,13 @@ if( ICONV_FOUND )
     target_link_libraries( exiv2lib PRIVATE Iconv::Iconv )
 endif()
 
+if( inih_FOUND )
```

Comment:
(related to previous comment) If the library is required we should check in the file `cmake/findDependencies.cmake` if the library is not found and fail. Afterwards, here we could remove the conditional.